themes/default/content/blog/slack-integration/index.md

```diff
@@ -0,0 +1,397 @@
+}
+```
+
+{{% /choosable %}}
+
+{{% choosable language csharp %}}
+
+```csharp
+using System.Collections.Generic;
+using System.Linq;
+using Pulumi;
+using PulumiService = Pulumi.PulumiService;
+
+return await Deployment.RunAsync(() =>
+{
+    var orgWebhook = new PulumiService.Webhook("orgWebhook", new()
+    {
+        Active = true,
+        DisplayName = "#my-org-activity-channel",
+        OrganizationName = "my-org",
+        PayloadUrl = "https://hooks.slack.com/services/...",
+        Format = PulumiService.WebhookFormat.Slack,
+        Filters = new[]
+        {
+            PulumiService.WebhookFilters.StackCreated,
+            PulumiService.WebhookFilters.StackDeleted,
+            PulumiService.WebhookFilters.DeploymentSucceeded,
+            PulumiService.WebhookFilters.DeploymentFailed,
+        },
+    });
+
+    var stackWebhook = new PulumiService.Webhook("stackWebhook", new()
+    {
+        Active = true,
+        DisplayName = "#my-stack-activity-channel",
+        OrganizationName = "my-org",
+        ProjectName = "webhook-examples",
+        StackName = "dev",
+        PayloadUrl = "https://hooks.slack.com/services/...",
+        Format = PulumiService.WebhookFormat.Slack,
+    });
+
+    return new Dictionary<string, object?>
+    {
+        ["orgWebhookName"] = orgWebhook.Name,
+        ["stackWebhookName"] = stackWebhook.Name,
+    };
+});
+```
+
+{{% /choosable %}}
+
+{{% choosable language java %}}
+
+```java
+package generated_program;
+
+import com.pulumi.Context;
+import com.pulumi.Pulumi;
+import com.pulumi.core.Output;
+import com.pulumi.pulumiservice.Webhook;
+import com.pulumi.pulumiservice.WebhookArgs;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Map;
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class App {
+    public static void main(String[] args) {
+        Pulumi.run(App::stack);
+    }
+
+    public static void stack(Context ctx) {
+        var orgWebhook = new Webhook("orgWebhook", WebhookArgs.builder()
+            .active(true)
+            .displayName("#my-org-activity-channel")
+            .organizationName("my-org")
+            .payloadUrl("https://hooks.slack.com/services/...")
+            .format("slack")
+            .filters(
+                "stack_created",
+                "stack_deleted",
+                "deployment_succeeded",
+                "deployment_failed")
+            .build());
+
+        var stackWebhook = new Webhook("stackWebhook", WebhookArgs.builder()
+            .active(true)
+            .displayName("#my-stack-activity-channel")
+            .organizationName("my-org")
+            .projectName("webhook-examples")
+            .stackName("dev")
+            .payloadUrl("https://hooks.slack.com/services/...")
+            .format("slack")
+            .build());
+
+        ctx.export("orgWebhookName", orgWebhook.name());
+        ctx.export("stackWebhookName", stackWebhook.name());
+    }
+}
+```
```

Comment:
Missing a closing `choosable` for java, which is messing up the formatting for the rest of the post.

Reply:
fixed this!